marshmallow_sqlalchemy/convert.py

```diff
@@ -206,7 +207,7 @@ def _add_column_kwargs(self, kwargs, column):
             kwargs['validate'].append(validate.OneOf(choices=column.type.enums))
 
         # Add a length validator if a max length is set on the column
-        if hasattr(column.type, 'length'):
+        if hasattr(column.type, 'length') and not issubclass(column.type.python_type, uuid.UUID):
```

Comment:
I'm not sure that skipping validation is the correct solution. How about using a separate Length validator for UUID objects?